thirdparty/drmingw_build_mingw.cmd

```diff
@@ -21,13 +21,15 @@ set MINGW64_FN=x86_64-6.3.0-release-win32-seh-rt_v5-rev1.7z
 set MINGW32_URL=https://sourceforge.net/projects/mingw-w64/files/Toolchains%%20targetting%%20Win32/Personal%%20Builds/mingw-builds/6.3.0/threads-win32/dwarf/%MINGW32_FN%/download
 set MINGW64_URL=https://sourceforge.net/projects/mingw-w64/files/Toolchains%%20targetting%%20Win64/Personal%%20Builds/mingw-builds/6.3.0/threads-win32/seh/%MINGW64_FN%/download
 
+REM  I downloaded these manually.
 for %%t in (32 64) do (
 	set FN=!MINGW%%t_FN!
 	set URL=!MINGW%%t_URL!
 
 	if not exist "%~dp0\mingw\!FN!" (
 		echo Downloading !FN!...
 		set DEST=%~dp0/mingw/!FN!
+                REM If the automagic download fails, download the files manually and put them in this directory.

```

Comment:
This file uses tabs for indentation, not spaces.

Reply:
Bummer.